src/departure-list/section-items/line.tsx

```diff
@@ -70,15 +70,15 @@ export function LineItem({
   const {t, language} = useTranslation();
 
   const {onMarkFavourite, existingFavorite, toggleFavouriteAccessibilityLabel} =
-    useOnMarkFavouriteDepartures(
-      {...group.lineInfo, id: group.lineInfo?.lineId},
-      quay,
-      stop,
-    );
+    useOnMarkFavouriteDepartures(quay, stop);
 
   if (hasNoDeparturesOnGroup(group)) {
     return null;
   }
+  const favouriteDepartureLine = {
+    ...group.lineInfo,
+    id: group.lineInfo?.lineId ?? '',
```

Comment:
I can't see here the full picture here, but are there any implications if the ID is '' or would be best to just ignore if undefined?

Reply:
I didn't spend much time on this because it only affects departures v1, which is soon to be removed anyway. I don't think it will happen very often, if ever, and if it does it looks like it will make the favorite-star not work properly.